requirements.txt

```diff
@@ -1,5 +1,5 @@
 git-lfs
-natcap.invest
+git+https://github.com/natcap/invest.git@main
```

Comment:
I think we want to undo this change so that `invest.users-guide@main` builds against `natcap.invest` installed from PyPI.